systems-management/nisysmgmt.yml

```diff
@@ -0,0 +1,2076 @@
+          required: [id, action]
+        required: [key_actions]
+
+  SystemPatchRequest:
+    description: System data that has to be assigned to the system.
+    title: System Patch Request
+    x-ni-sparse: true
+    type: object
+    properties:
+      keywords:
+        description: The new keywords assigned to the system.
+        type: array
+        items:
+          type: string
+        example:
+          - test
+          - vm
+      properties:
+        description: The new properties assigned to the system.
+        type: object
+        additionalProperties:
+          type: string
+        example:
+          owner: admin
+      alias:
+        description: The new  alias of the system.
+        type: string
+        example: testVM
+
+  QueryAvailablePackagesRequest:
+    title: Get Available Packages Request
+    description: Request to get the available packages from the feeds specified in the request.
+    type: object
+    properties:
+      return_duplicates:
+        description: A boolean representing whether to return packages of equal version for each architecture.
+        type: boolean
+        default: false
+        example: false
+      system_configurations:
+        description: An array of system configurations.
+        type: array
+        items:
+          type: object
+          title: System Configuration
+          properties:
+            feed_uris:
+              type: array
+              description: An array of strings representing feed URIs a system(s) has configured.
+              items:
+                type: string
+              example:
+                - http://download.ni.com/ni-linux-rt/feeds/SystemLink/18.5/x64/
+                - http://download.ni.com/ni-linux-rt/feeds/SystemLink/salt/18.5/x64/x64/           
+            os_architectures:
+              type: array 
+              description: An array of strings representing OS architectures supported by a system(s) in descending priority.
+              items:
+                type: string
+              example:
+                - x64
+                - core2-64
+                - x86_64
+                - noarch
+                - any
+                - all
+        required: [feed_uris]
+    required: [system_configurations]
+
+  PackageResponse:
+    title: Get Available Packages Response
+    description: A list with the available packages.
+    type: object
+    properties:
+      arch:
+        type: string
+        description: The architecture of the package.
+        example: windows_x64
+      breaks:
+        type: array
+        description: An array containing information about other packages this package breaks.
+        items:
+          type: string
+          description: The name of a package that this package breaks. This also contains information about the version of the package.
```

Comment:
could we remove "that" (i.e. "that this package breaks" --> "package this package breaks")? This removal request applies to the following descriptions that have "that this" in them.